src/main.sh

```diff
@@ -96,6 +97,10 @@ function main {
       installTerraform
       terraformApply "${*}"
       ;;
+    output)
+      installTerraform
+      terraformOutput "${*}"
```

Comment:
The quotes need to be absent here. See #111 for why.
```suggestion
      terraformOutput ${*}
```